src/components/Form.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function Form({stateForm, stateFormSetter, formLabel, formName, formID, formType}) {
```

Comment:
Ik zou een andere naam kiezen voor dit component. Het is in feite geen form wat je hier teruggeeft 😉